src/hd.rs

```diff
@@ -412,27 +350,17 @@ impl KeyOrigin {
     pub fn from_label(s: &str) -> Option<Self> {
         let parts: Vec<&str> = s.splitn(3, '/').collect();
         match (parts.get(0), parts.get(1), parts.get(2)) {
-            (Some(&LABEL_PREFIX), Some(parent), Some(index)) => Some(KeyOrigin::Derived(
-                Fingerprint::from(&hex::decode(parent).ok()?[..]),
+            // FIXME: LABEL_PREFIX is missing for me ...
+            //(Some(&LABEL_PREFIX), Some(parent), Some(index)) => Some(KeyOrigin::Derived(
```

Comment:
This is a bug. The label prefix isn't getting filled in. Seems like [this line](https://github.com/justinmoon/bwt/blob/9e1f9ad0781d5de5c6ec5505c289cea3271d9d68/src/hd.rs#L339) isn't getting exercised.